src/site/pages/upgrade-guide.md

```diff
@@ -0,0 +1,223 @@
+# PatternFly 4 Upgrade Guide
+
+Upgrading your environment from PatternFly 3 to PatternFly 4.
+
+**Important**
+
+When upgrading from a PatternFly 3 environment to a PatternFly 4 environment, you will need to do the following:
+
+- Verify that your application does not need IE 11 support. If it does, you cannot upgrade to PatternFly 4.
+- Check your build process. The steps for basic HTML, Gulp and Webpack differ and have different requirements.
+- Determine if you need to run PatternFly 3 and PatternFly 4 together. If you do, you will need to update the following configuration in the `src/patternfly/utilities/variables.scss` file to state `true`.
+
+```scss
+$pf-global--load-pf-3: true !default;
+```
+
+
+## Installing PatternFly 4
```

Comment:
Installing PatternFly 4
Run the following commands to install:

Configuring your project(?)

1. Navigate to the patternfly-next package you just installed (do we want to change the name of this to patternfly-4?) and find the PatternFly 4 CSS stylesheet, `patternfly.css.` This file contains all of the updated PatternFly 4 styles. 
2. Copy `patternfly.css` to your project's CSS directory. 
3. (Trying to figure out a way to say this clearly...)In your HTML file, add the following line to the bottom of your list of CSS files to link to your new stylesheet:"

--Obviously if any of this is wrong or misleading let me know :)